static/js/main.js

```diff
@@ -214,15 +213,15 @@
     switch (os) {
       case 'Mac':
         versionIntoHref(buttons, 'node-%version%.pkg')
-        downloadHead[text] = dlLocal + ' macOS (x64)'
+        downloadHead.textContent = dlLocal + ' macOS (x64)'
```

Comment:
@targos should this button show ARM64 too for Node.js >= 16? If so, we'll need to adapt it later.

Reply:
Where is this?

Reply:
This is the text above the buttons show in the homepage.

Reply:
I see. That case is a bit ambiguous, because the text applies to both buttons (LTS and Current) but ARM64 is only available for Current.